### PR TITLE
Some developer notes.

### PR DESCRIPTION
**Pull request by paulp**

### CODE REVIEW

File: doc/fuse.txt [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Fuse/system data structures.
```
Comment (EECOLOR) [inline doc/fuse.txt]: I think it would be nice if this was an `.md` file with some code blocks.

File: doc/kernel.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Kernel protocol
```
Comment (EECOLOR) [inline doc/kernel.txt]: Maybe an `.md` file?

Can you describe in a single sentence where this is used in the library?

File: doc/serve.txt [hunk 3]
```diff
@@ -0,0 +1,79 @@
+# Serve distillation.
+#
```
Comment (EECOLOR) [inline doc/serve.txt]: Can you add a single line with a discription of what this is used for?

It's unclear to me (without any effort to try and find out) if these are interfaces that are part of this library or an external one.

**Comment by paulp** [comment]
@EECOLOR okay please give another look.

**Comment by EECOLOR** [comment]
:+1:

**Pull request merged**